docs/operator-manual/argocd-cmd-params-cm.yaml

```diff
@@ -20,6 +20,13 @@ data:
   # Open-Telemetry collector address: (e.g. "otel-collector:4317")
   otlp.address:
 
+  # List of additional namespaces where applications may be created in and
+  # reconciled from. The namespace where Argo CD is installed to will always
+  # be allowed.
+  #
+  # Feature state: Beta
+  application.namespaces: ns1, ns2, ns3
```

Comment:
Are the spaces okay? Just glanced at `StringsFromEnv`, and it looks like the last two namespaces might be parsed as `ns2 ` and `ns3 `.

Reply:
yup, I was just about to say the same. can't have spaces in there 😄

Reply:
I think `StringsFromEnv` should strip leading and trailing white space, so I submitted #10275 :)

Reply:
Merged!